.github/workflows/benchmarks.yml

```diff
@@ -40,10 +40,12 @@ jobs:
           KUROBAKO: ./kurobako
         run: ./benchmark/runner.sh six-hump-camel ./kurobako-report.json
       - name: Plot kurobako result
-        uses: c-bata/github-actions-kurobako/plot@v1
+        uses: c-bata/github-actions-kurobako/plot@master
         id: kurobako-plot
         with:
           report-json-path: './kurobako-report.json'
+          error-bar: 'true'
+          ymin: '-1.05'
```

Comment:
We cannot pass negative value due to the problem of command line arguments parser.